es_qat_wrapper/8.0.0/elasticsearch/test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -0,0 +1,2110 @@
+/*
```

Comment:
We should use ```patch``` command to generate patch instead of copying the entire file here.